src/main/docker/configs/bucketeer.properties.tmpl

```diff
@@ -21,6 +21,7 @@ http.max.queue.size = $HTTP_MAX_QUEUE_SIZE
 bucketeer.max.source.file.size = $SOURCE_MAX_FILE_SIZE
 s3.uploader.instances = $S3_UPLOADER_INSTANCES
 s3.uploader.threads = $S3_UPLOADER_THREADS
+s3.max.requests = $S3_MAX_REQUESTS
```

Comment:
This might be the problem. I set the envs, but it doesn't propagate during runtime. This should fix it

Reply:
The default for this is 10 (in the code). I'd suggest we set to 1 for testing.